layers/+tools/shell/funcs.el

```diff
@@ -31,6 +31,12 @@ and move your focus to it; to switch the current buffer view, use
   (let ((default-directory (projectile-acquire-root)))
     (call-interactively 'spacemacs/default-pop-shell)))
 
+(defun spacemacs//projectile-run-shell-in-root (&optional shell)
+  "Invoke `multi-term' in the project's root."
+  (interactive)
+  (projectile-with-default-dir (projectile-project-root)
+    (call-interactively (intern-soft (or shell shell-default-shell)))))
```

Comment:
`shell` is already known to be a symbol, there is no need to call `intern-soft`

Reply:
Also, since this is an internal private function, and the `shell` argument is never used, I think we can just get rid of it.

Reply:
Arguments was removed, and `intern-soft` also was removed.